client/landing/stepper/declarative-flow/internals/steps-repository/use-my-domain/index.tsx

```diff
@@ -1,7 +1,12 @@
 /* eslint-disable wpcalypso/jsx-classname-namespace */
 import { useDispatch } from '@wordpress/data';
 import { getQueryArg } from '@wordpress/url';
-import { StepContainer } from 'calypso/../packages/onboarding/src';
+import {
+	DESIGN_FIRST_FLOW,
+	START_WRITING_FLOW,
+	StepContainer,
+	isStartWritingFlow,
+} from 'calypso/../packages/onboarding/src';
```

Comment:
```suggestion
import {
	DESIGN_FIRST_FLOW,
	START_WRITING_FLOW,
	StepContainer,
	isStartWritingFlow,
} from '@automattic/onboarding';
```

Looks a bit more prettier

Reply:
Nice! [Fixed here](https://github.com/Automattic/wp-calypso/pull/77222/commits/fb3ce06b5fffb5d2cf9c181171c220dc5ac26719)